src/v/coproc/script_context.cc

```diff
@@ -336,22 +338,17 @@ script_context::write_checked(
   model::term_id last_term,
   model::record_batch_reader reader) {
     using rt_val = std::variant<write_response, model::term_id>;
-    using consumption_result = std::tuple<
-      bool,
-      std::optional<model::term_id>,
-      ss::future<storage::append_result>>;
-    const storage::log_append_config write_cfg{
-      .should_fsync = storage::log_append_config::fsync::no,
-      .io_priority = ss::default_priority_class(),
-      .timeout = model::no_timeout};
+    using consumption_result = std::
```

Comment:
why not extract this as a `POD` like `struct <something>_consumption_result {}`

Reply:
addressed a comment below